apps/app/src/test/scala/org/lfdecentralizedtrust/splice/util/SvTestUtil.scala

```diff
@@ -191,6 +196,93 @@ trait SvTestUtil extends TestCommon {
     )
   }
 
+  def modifyValidatorLicensesWithVoting(
+      svToCreateVoteRequest: SvAppBackendReference,
+      svsToCastVotes: Seq[SvAppBackendReference],
+      changes: Seq[ValidatorLicenseChange],
+      checkClue: String = "validator license modifications have been applied",
+  )(
+      verifyResult: => Unit
+  )(implicit
+      ec: ExecutionContextExecutor
+  ): Unit = {
+    val action = new ARC_DsoRules(
+      new SRARC_ModifyValidatorLicenses(
+        new ValidatorLicensesModification(changes.asJava)
+      )
+    )
+
+    def onlyModifyValidatorLicensesVoteRequests(
+        voteRequests: Seq[Contract[VoteRequest.ContractId, VoteRequest]]
+    ) =
+      voteRequests.filter {
+        _.payload.action match {
+          case arc: ARC_DsoRules =>
+            arc.dsoAction match {
+              case _: SRARC_ModifyValidatorLicenses => true
+              case _ => false
+            }
+          case _ => false
+        }
+      }
+
+    actAndCheck(timeUntilSuccess = 60.seconds)(
+      "Voting on validator license modification", {
+        val (_, voteRequest) = actAndCheck(
+          "Creating vote request",
+          eventuallySucceeds() {
+            svToCreateVoteRequest.createVoteRequest(
+              svToCreateVoteRequest.getDsoInfo().svParty.toProtoPrimitive,
+              action,
+              "url",
+              "description",
+              svToCreateVoteRequest.getDsoInfo().dsoRules.payload.config.voteRequestTimeout,
+              None,
+            )
+          },
+        )(
+          "vote request has been created",
+          _ =>
+            onlyModifyValidatorLicensesVoteRequests(
+              svToCreateVoteRequest.listVoteRequests()
+            ).loneElement,
+        )
+
+        if (svsToCastVotes.nonEmpty) {
+          loggerFactory.suppressErrors {
```

Comment:
That's a very heavy hammer, which risks hiding actual errors. Why is this required?

Reply:
Removed this